io/io/inc/ROOT/RIoUring.hxx

```diff
@@ -0,0 +1,64 @@
+/*************************************************************************
+ * Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_RIoUring
+#define ROOT_RIoUring
+
+#include <cstring>
+#include <stdexcept>
+
+#include <liburing.h>
+#include <liburing/io_uring.h>
+
+#include "TError.h"
+#include <ROOT/RLogger.hxx>
```

Comment:
Where is `RLogger` used?

Reply:
Another remnant from the past, thank you.